metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/v3/OpenAPIV3Generator.java

```diff
@@ -823,8 +828,7 @@ private static PathItem buildSingleEntityAspectPath(
                                 new Schema()
                                     .$ref(
                                         String.format(
-                                            "#/components/schemas/%s%s",
-                                            upperFirstAspect, ASPECT_REQUEST_SUFFIX)))));
+                                            "#/components/schemas/%s", upperFirstAspect)))));
```

Comment:
This does not feel like the correct change, it's not inline with how the other APIs are structured and significantly changes the contract. I don't think I'm comfortable signing off on this for it to just be flipped back once David comes back. I believe the right fix here would be to modify the toUpsertItem method to pull the top level field that contains the actual aspect so:

```
  protected ChangeMCP toUpsertItem(
      @Nonnull AspectRetriever aspectRetriever,
      Urn entityUrn,
      AspectSpec aspectSpec,
      Boolean createIfNotExists,
      String jsonAspect,
      Actor actor) throws URISyntaxException, JsonProcessingException {
    JsonNode jsonNode = objectMapper.readTree(jsonAspect);
    String aspectJson = jsonNode.get("value").toString();
    return ChangeItemImpl.builder()
        .urn(entityUrn)
        .aspectName(aspectSpec.getName())
        .changeType(Boolean.TRUE.equals(createIfNotExists) ? ChangeType.CREATE : ChangeType.UPSERT)
        .auditStamp(AuditStampUtils.createAuditStamp(actor.toUrnStr()))
        .recordTemplate(
            GenericRecordUtils.deserializeAspect(
                ByteString.copyString(aspectJson, StandardCharsets.UTF_8),
                GenericRecordUtils.JSON,
                aspectSpec))
        .build(aspectRetriever);
  }
```

Reply:
That works for me too 👍 
I wanted to minimize the change while David was out but this will fix it as well 
I'll go ahead and make this change instead